tests/test_integration/test_tag.py

```diff
@@ -299,16 +297,8 @@ def test_tag_sort_by_tag_date(repo, create_config, message):
 
     for tag in tags_to_create:
         create_tag(repo, tag, message=message, commit=commits[tag])
-        time.sleep(1)
 
-    if message:
-        # the result is not stable because latest tag (by creation time)
-        # has nothing in common with commit creation time
-        assert "1.1.10" in get_version(repo)
-    else:
-        assert get_version(repo).startswith("1.1")
-        # the result is totally random because annotaged tags have no such field at all
-        # https://github.com/dolfinus/setuptools-git-versioning/issues/23
+    assert get_version(repo).startswith("1.1")
```

Comment:
Please revert this. `GIT_COMMITER_DATE` has nothing in common with tag creation date

Reply:
K - IIRC, the tests weren't passing with the GIT_COMMITTER_DATE set, but since it was set incorrectly, I'll see what happens when it's fixed.

Reply:
Yep, works fine with the ``GIT_COMMITTER_DATE`` fix and the revert.